### PR TITLE
chore: pkg imported more than once

### PR DESCRIPTION
**Pull request by testwill**
packages is being imported more than once (ST1019)

### CODE REVIEW

**Review by dylandreimerink — APPROVED** [review]

File: pkg/bgpv1/test/adverts_test.go [hunk 1]
```diff
@@ -9,7 +9,6 @@ import (
 	"time"
 
 	corev1 "github.com/cilium/cilium/pkg/k8s/slim/k8s/api/core/v1"
-	slim_metav1 "github.com/cilium/cilium/pkg/k8s/slim/k8s/apis/meta/v1"
 	slimv1 "github.com/cilium/cilium/pkg/k8s/slim/k8s/apis/meta/v1"
```
Comment (tklauser) [inline pkg/bgpv1/test/adverts_test.go:12]: I'd keep `slim_metav1` here and drop `slimv1` instead, as the name is more specific.

**Review by tklauser — CHANGES_REQUESTED** [review]
I assume you found these using some linter (probably `staticcheck`?). Could we also enable that check in our CI so we catch these types of issues in the future?

**Review by giorio94 — APPROVED** [review]

File: pkg/hubble/parser/threefour/parser.go [hunk 2]
```diff
@@ -15,7 +15,6 @@ import (
 	"github.com/sirupsen/logrus"
 	"google.golang.org/protobuf/types/known/wrapperspb"
 
-	"github.com/cilium/cilium/api/v1/flow"
 	pb "github.com/cilium/cilium/api/v1/flow"
```
Comment (youngnick) [inline pkg/hubble/parser/threefour/parser.go:16]: I think it's better to keep the unaliased version here maybe? `pb` is a pretty generic name that doesn't give much clue about what the package is actually doing.

**Review by youngnick — COMMENTED** [review]
One extra thing from me, but I agree that we should add whatever linter was used here to the standard set of checks.

**Comment by ti-mo** [comment]
I need to echo the others, this PR should come with a linter so this gets caught in the future.

**Comment by ldelossa** [comment]
@testwill if you are still interested in merging this PR, can you please rebase and push back up?

**Review by viktor-kurchenko — APPROVED** [review]
LGTM.

**Review by gandro — APPROVED** [review]
The changes in my code owners look good to me. But the commit message needs to be changed.

Approving for just my code owners part, defaulting Tobias for questions around CI.

**Review by nathanjsweet — APPROVED** [review]

**Comment by tommyp1ckles** [comment]
@testwill thanks for the PR,  the changes are pretty straightforward but could you add a more accurate description of the changes in the commit message?

**Review by christarazi — APPROVED** [review]

**Comment by christarazi** [comment]
@testwill Please update the commit msg to have a more description title and msg. The title of the PR works just fine.

File: pkg/hubble/parser/threefour/parser.go (hunk 2 above)
Comment (tommyp1ckles) [reply pkg/hubble/parser/threefour/parser.go:16]: Unresolving this as it appears to still be using "pb", I agree with nick - pb might get confusing given there are multiple protobuffer libraries.

**Comment by tommyp1ckles** [comment]
@testwill 

Looks good, just waiting for one point that was missed regarding the flow pb library.

**Comment by squeed** [comment]
@testwill needs rebase.

**Comment by ldelossa** [comment]
@testwill can you please rebase this PR if you still intend of merging?

**Comment by ti-mo** [comment]
@testwill Sorry, this code has a bit of churn, you'll need to rebase again.

File: pkg/bgpv1/test/adverts_test.go (hunk 1 above)
Comment (joestringer) [reply pkg/bgpv1/test/adverts_test.go:12]: This wasn't yet addressed, but it was marked as resolved.

**Comment by joestringer** [comment]
I think that in principle this has been approved by most codeowners, there's just a couple of specific comments where the other import is preferred since it's more descriptive. Looks like it needs a rebase too.

I agree that adding a linter would be a good step, though I don't see it as a strict requirement to merge this PR. I've filed https://github.com/cilium/cilium/issues/27676 to follow up on that request.

I’ve marked this PR as draft while you work through the feedback in the PR above. When you’re ready to continue with review & testing of the PR, please click “Ready for Review” at the bottom of the page.

**Pull request closed without merging**